### ISSUE TITLE
VGG16 pre-trained model's accuracy is a little Lower than expected

### ISSUE THREAD

**Issue by yiyangjiang** [tensorflow/models#1833]
I evaluated the vgg_16 model on imagenet validation set with eval_image_classifier.py.

```
python eval_image_classifier.py --alsologtostderr \
    --checkpoint_path=/home/jyy/ckpt/vgg_16.ckpt \
    --dataset_dir=/media/jyy/F/tf_imagenet \
    --dataset_name=imagenet \
    --dataset_split_name=validation \
    --model_name=vgg_16 \
    --labels_offset=1
```

The val data was generated using script [https://github.com/tensorflow/models/blob/master/inception/inception/data/download_and_preprocess_imagenet.sh](url)

The ckpt file is downloaded from [http://download.tensorflow.org/models/vgg_16_2016_08_28.tar.gz](url)

The final top 1 accuracy is 70.93% which is a little lower than 71.5% in the table.
While the top 5 accuracy is 89.82% which is right.
VGG 16|Code|vgg_16_2016_08_28.tar.gz|71.5|89.8

I wish to know if there is anything wrong.

Thanks!

UPDATE:
I also evaluated ResNet_V1_152 and the accuracy is right.
ResNet V1 152|Code|resnet_v1_152_2016_08_28.tar.gz|76.8|93.2
```
2017-07-02 01:19:57.291579: I tensorflow/core/kernels/logging_ops.cc:79] eval/Accuracy[0.76798]
2017-07-02 01:19:57.292239: I tensorflow/core/kernels/logging_ops.cc:79] eval/Recall_5[0.9318]
```

**Comment by cy89** [comment]
Have you retried your run? Possibly a number of times? It sounds as if you have just done a single run, and variation in run-to-run accuracy is expected.

**Comment by yiyangjiang** [comment]
@cy89 I tried several times and the result is the same. Top 1 accuracy is always 70.93%. I don't know what's the problem.

**Comment by YaoZhongtian** [comment]
I got the same result as @yiyangjiang, and I also don't know why.

**Comment by jetsmith** [comment]
@yiyangjiang i got 70.98% which is also lower than 71.5%, maybe the different preprocessing, i use the prep script:
[https://github.com/tensorflow/models/blob/master/research/slim/preprocessing/vgg_preprocessing.py](url).
have you solve it?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by amahendrakar** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
